src/main/java/com/assignment1/item/TaxCalculator.java

```diff
@@ -0,0 +1,46 @@
+package com.assignment1.item;
+
+import com.assignment1.Type;
+
+/**
+ * Class to implement Tax Calculation logic.
+ */
+@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
```

Comment:
can you check pmd version, this is deprecated in latest version

Reply:
Yes, checked with the updated version. Works fine now.